Source/WebKit/WebProcess/Network/webrtc/LibWebRTCNetwork.cpp

```diff
@@ -111,18 +111,33 @@ void LibWebRTCNetwork::dispatch(Function<void()>&& callback)
 }
 
 #if USE(LIBWEBRTC)
+static rtc::EcnMarking convertToWebRTCEcnMarking(RTC::Network::EcnMarking ecn)
+{
+    switch (ecn) {
+    case RTC::Network::EcnMarking::kNotEct:
+        return rtc::EcnMarking::kNotEct;
+    case RTC::Network::EcnMarking::kEct1:
+        return rtc::EcnMarking::kEct1;
+    case RTC::Network::EcnMarking::kEct0:
+        return rtc::EcnMarking::kEct0;
+    case RTC::Network::EcnMarking::kCe:
+        return rtc::EcnMarking::kCe;
+    }
```

Comment:
Let's add `ASSERT_NOT_REACHED();`

Reply:
@youennf  Thanks for the review. Updated.